.travis.yml

```diff
@@ -1,13 +1,35 @@
 sudo: false
 language: node_js
 script:
-  - npm run build
   - npm run test:ci
 node_js:
+  - 0.10
+  - 2
+  - 4
+  - 6
   - 8
   - 10
   - 12
 
+before_install:
+  - npm config set depth 0
+before_script: >
+  node_version=$(node -v);
```

Comment:
You can get the major directly with `node -v | cut -d . -f 1 | tail -c +2`

Reply:
If testing against 0.12, IIRC, the 0 isn't shown.